app/assets/javascripts/discourse/tests/acceptance/wizard-test.js

```diff
@@ -75,6 +85,20 @@ acceptance("Wizard", function (needs) {
     await click(".wizard-container__link.back");
     assert.ok(exists(".wizard-container__step-title"));
     assert.ok(exists(".wizard-container__button.next"));
-    assert.ok(!exists(".wizard-prev"));
+    await click(".wizard-container__button.next");
+
+    // server validation fail
+    await fillIn("input#company_name", "Server Fail");
+    await click(".wizard-container__button.jump-in");
+    assert.ok(exists(".invalid #company_name"));
+    assert.ok(exists(".wizard-container__field .error"));
```

Comment:
I know the existing tests do not add a short description to the assertion but it'll be nice if we start doing it because it helps the read to understand what the assertion is trying to do.